client/reader/_style.scss

```diff
@@ -23,7 +23,6 @@
 .is-reader-page .follow-button {
 	border: 0;
 	border-radius: 0;
-	float: right;
 }
```

Comment:
- [ ] Make sure Follow(ing) buttons in other pages are still floated right - ie Tags page.